### ISSUE TITLE
Hiding word suggestions on iOS keyboard?

### ISSUE THREAD

**Issue by julesmoretti** [driftyco/ionic-plugin-keyboard#106]
Hello there:

Thank you for this great plugin.

Despite the two following lines:
```
cordova.plugins.Keyboard.hideKeyboardAccessoryBar(true);
cordova.plugins.Keyboard.disableScroll(false);
```
Which gets rid of the `done` button and the `left` & `right` arrow successfully.

The word suggestion still shows up. Is there a way to remove those?

Thank you.

**Comment by mrzmyr** [comment]
+1

**Comment by Hirbod** [comment]
Just do it in HTML like I did.


```html
<input 
    autocorrect="off"
    autocapitalize="off">
```

**Comment by vmeng** [comment]
@Hirbod  How to remove `left` and `right` arrow?

**Comment by Hirbod** [comment]
cordova.plugins.Keyboard.hideKeyboardAccessoryBar

**Comment by vmeng** [comment]
@Hirbod Yes, this can hide left and right arrow, but this also hide done button. I just want hide left and right arrow, keep done button show.

**Comment by Hirbod** [comment]
As far as I know, this is not possible. You need to write your own keyboard then.

**Comment by rejois** [comment]
@Hirbod (or anyone else) - instead of hiding the left and right arrows, do you know of a way to disable their functionality instead? (Without writing our own keyboard)  A usecase could be when we want to prevent a user from moving back and forth from a singly form field with the < and > arrows?
Thanks!

**Comment by SopoB** [comment]
@Hirbod did you got any solution or any updates using @rejois suggestion?
Please post. 
I got similar issue and i also wants done button only without arrows in accessary bar

**Comment by frapporti** [comment]
Have you tried using separated inputs instead of a form?
